apps/vs-code-designer/src/app/commands/funcCoreTools/validateFuncCoreToolsIsLatest.ts

```diff
@@ -49,7 +50,7 @@ export async function validateFuncCoreToolsIsLatest(): Promise<void> {
           const result: MessageItem = await context.ui.showWarningMessage(message, selectUninstall, DialogResponses.dontWarnAgain);
 
           if (result === selectUninstall) {
-            await executeOnFunctions(uninstallFuncCoreTools, context, packageManagers);
+            await executeOnFunctions(uninstallFuncCoreTools, context, context, packageManagers);
```

Comment:
It seems that with the update of uninstallFuncCoreTools file, we don't need to wrap it with the executeOnFunctions function, therefore there will no need to add an additional context.